src/index.ts

```diff
@@ -117,14 +132,21 @@ const withSpidAuthMiddleware = (
  * Apply SPID authentication middleware
  * to an express application.
  */
+// tslint:disable-next-line: parameters-max-number
 export function withSpid(
   appConfig: IApplicationConfig,
   samlConfig: SamlConfig,
   serviceProviderConfig: IServiceProviderConfig,
   redisClient: RedisClient,
   app: express.Express,
   acs: AssertionConsumerServiceT,
-  logout: LogoutT
+  logout: LogoutT,
+  logCallback?: (
```

Comment:
You need pass this function even to `makeSpidStrategy`

Reply:
Yes, I'll make changes!